driver_scripts/driver_grid.hera.sh

```diff
@@ -106,10 +106,14 @@ export soil_type_src="bnu.v3.30s" #  Soil type data.
                                 # For Beijing Norm. Univ. data
                                 # 1) "bnu.v3.30s" for global 30s data.
 
+# choose dataset sources for lakefrac & lakedepth so that lake_data_srce=LakeFrac_LakeDepth; 
+# available options are 'MODISP_GLDBV3', 'MODISP_GLOBATHY', 'VIIRS_GLDBV3', 'VIIRS_GLOBATHY' & 'GLDBV3'
+export lake_data_srce=MODISP_GLDBV3 
+
 if [ $gtype = uniform ]; then
   export res=96
-  export add_lake=false        # Add lake frac and depth to orography data.
-  export lake_cutoff=0.20      # lake frac < lake_cutoff ignored when add_lake=T
+  export add_lake=true         # Add lake frac and depth to orography data.
+  export lake_cutoff=0.50      # lake frac < lake_cutoff ignored when add_lake=T
```

Comment:
add line after this:
export gridcell_lake = 1  # 1:only consider grid cell lakes; 0: consider fractional lakes

Reply:
Good idea. Is "gridcell_lake" same as "binary_lake"?

Reply:
yes, maybe binary_lake is a better name.

Reply:
maybe the comment should say "# 1:only consider full grid cell lakes; 0: consider fractional lakes"

Reply:
or "# 1:only output full grid cell lakes; 0: output fractional lakes"